pretender.js

```diff
@@ -21,6 +21,7 @@ function Pretender(maps){
   this.handledRequests = [];
   this.passthroughRequests = [];
   this.unhandledRequests = [];
+  this.requestRefs = [];
```

Comment:
This existing code doesn't tend to use abbreviations. This ref says "Yellow Card"

![](http://media.philly.com/images/referee2.jpg)